src/data/pequiv.lean

```diff
@@ -144,7 +144,7 @@ has_left_inverse.injective
       classical,
       cases hfx : f x,
       { have : x = a₂, from not_imp_comm.1 (h x) (hfx.symm ▸ by simp), simp [this] },
-      { simp only [hfx], rw [(eq_some_iff f).2 hfx], refl }
+      { simp only [], rw [(eq_some_iff f).2 hfx], refl }
```

Comment:
Could this be replaced with `dsimp only`?

Reply:
Seems so